Casks/s/soundflowerbed.rb

```diff
@@ -7,5 +7,7 @@
   desc "Taps into Soundflower channels and route them to an audio device"
   homepage "https://github.com/mLupine/SoundflowerBed"
 
+  deprecate! date: "2024-15-16", because: :discontinued
```

Comment:
```suggestion
  deprecate! date: "2024-05-16", because: :discontinued
```

Little typo snuck in here

Reply:
Thanks!